README.md

```diff
@@ -1 +1,179 @@
+        <th>Default</th>
+        <th>Description</th>
+      </tr>
+      <tr>
+        <td>cluster.name</td>
+		<td>ELASTICSEARCH_CLUSTER_NAME</td>
+        <td>docker-cluster</td>
+        <td>A node can only join a cluster when it shares its cluster.name with all the other nodes in the cluster</td>
+      </tr>
+      <tr>
+        <td>ES_JAVA_OPTS</td>
+		<td>ES_JAVA_OPTS</td>
+        <td>-Xmx256m -Xms256m</td>
+        <td>Environment variable to set heap size <br />e.g. to use 1GB use ES_JAVA_OPTS="-Xms1g -Xmx1g"</td>
+      </tr>
+      <tr>
+        <td>ports</td>
+		<td>ELASTICSEARCH_HTTP_PORT</td>
+        <td>9200</td>
+        <td>HTTP Port used for RESTful API</td>
+      </tr>
+      <tr>
+        <td>ports</td>
+		<td>ELASTICSEARCH_NETWORK_PORT</td>
+        <td>9300</td>
+        <td>Network Communication Port used for Java API, the Elasticsearch transport protocol and Cluster Communications</td>
+      </tr>
+    </table>
+
+3. Disable X-Pack  
+X-Pack is provided as default within the Elasticsearch image.  To disable X-Pack as currently no license is held for the use of X-Pack set the following parameters.
+    <table>
+      <tr>
+		<th>Docker Compose Variable</th>
+        <th>Value</th>
+        <th>Description</th>
+      </tr>
+      <tr>
+        <td>xpack.security.enabled</td>
+        <td>false</td>
+        <td>Set to false to disable X-Pack security</td>
+      </tr>
+      <tr>
+        <td>xpack.monitoring.enabled</td>
+        <td>false</td>
+        <td>Set to false to disable X-Pack monitoring</td>
+      </tr>
+      <tr>
+        <td>xpack.graph.enabled</td>
+        <td>false</td>
+        <td>Set to false to disable X-Pack graph</td>
+      </tr>
+      <tr>
+        <td>xpack.watcher.enabled</td>
+        <td>false</td>
+        <td>Set to false to disable X-Pack watcher</td>
+      </tr>
+    </table> 
+	For more information on X-Pack:
+	* https://www.elastic.co/guide/en/x-pack/current/xpack-settings.html
+	* https://www.elastic.co/guide/en/x-pack/current/installing-xpack.html#xpack-enabling
+
+4. Deploy the services  
+First navigate to the folder where you have downloaded the files to and then run one of the following commands, depending on whether you are using Docker Compose or Docker Stack:
+
+    <table>
+      <tr>
+        <td><b>Docker Compose</b></td>
+        <td>
+			docker-compose up  <small>(docker-compose defaults to use a file called <i><b>docker-compose.yml</b></i>)</small><br />
+			docker-compose up -d <small>(<i><b>-d</b></i> flag is for "detached mode" i.e. run containers in the background)</small>
+		</td>
+      </tr>
+      <tr>
+        <td><b>Docker Stack</b></td>
+        <td>docker stack deploy --compose-file=docker-compose.yml auditservicedemo</td>
+      </tr>
+    </table>
+
+5. Check the Health of Elasticsearch  
+The health of the Elasticsearch container and / or cluster can be inspected by issuing the following command:    
+	`curl http://localhost:9200/_cat/health`    
```

Comment:
- I'm not sure if we should assume that the Docker Host is always on `localhost`.  Did Andy say to write this specifically for Docker for Windows users?
- We could replace `9200` with `${ELASTICSEARCH_HTTP_PORT:-9200}` so that if they're using a different port the command will still be correct.